priv/templates/nimble_template/assets/css/mixins/_text-ellipsis.scss

```diff
@@ -0,0 +1,20 @@
+@mixin text-ellipsis(
+  $lines: 2,
+  $font-size: $font-size-base,
+  $line-height: $line-height-base
```

Comment:
These variables come from bootstrap, shall we add them into the `_variables.scss` in case the user doesn't put bootstrap in the project? 🤔

Reply:
I was thinking about that @bterone but I decide to not add that, because in case someone really would like to use that they will need to decide their `base font-size` and `base line-height` on their end.

Also this is the NimbleCSS addon, so the user is mostly coming from Nimble team :D and I'm assuming most of our project will use Boostrap so far :D 

But if you still think we need to define a `default value` so what value should we define as the base 🤔

Reply:
For default values, `font-size` could be the same as what we defined in `_sizing.scss` `16px` 
and line height could be `1.5` from bootstrap's [variables](https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L558) for now, what do you think? 🤔 

Since this addon could also be used by the community, they might wonder why the mixin isn't working out of the box and they'd need to manually assign variable values 😱 

On a separate topic, there's also the [!default](https://sass-lang.com/documentation/variables#default-values) flag we could experiment with, perhaps it could be useful for this case 💭

Reply:
added in https://github.com/nimblehq/elixir-templates/pull/154/commits/0a69723890bc1ce079f220a54baf41673461f650 @bterone